src/image_builder.py

```diff
@@ -170,6 +170,7 @@ def image_builder(buildspec):
             stage=constants.PRE_PUSH_STAGE,
             context=context,
             additional_tags=[image_tag],
+            target=image_config.get("target")
```

Comment:
Where is the target configured for `e3`

Reply:
Target doesn't have to be explicitly mapped to "e3" or "sagemaker" - it just corresponds to a build target in a multistage dockerfile. If I had a target called "e3" for instance, I'd just specifly `target: e3` in my buildspec